[IMP] developer: JavaScript tutorial: enhance chapter 1 (#3177)
odoo/documentation: content/developer/howtos/discover_js_framework/01_components.rst

=======================

As a first exercise, let us implement a counter in the `Playground` component located in

> :file:`owl_playground/static/src/`. To see the result, you can go to the `owl_playground/playground`
> route with your browser.

Reviewer: Let's make it clearer that it's a route.
```suggestion
:file:`owl_playground/static/src/`. To see the result, you can go to the `/owl_playground/playground`
route with your browser.
```

> :file:`owl_playground/static/src/`. To see the result, you can go to the `owl_playground/playground`
> route with your browser.

Reviewer: No need to apply this suggestion, but it might help you write more concise instructions in your future docs. Avoid "you can go", "you will see", etc.
```suggestion
:file:`owl_playground/static/src/`. Visit the route `/owl_playground/playground` in your browser to see the result.
```
